tests/core/test_solve.py

```diff
@@ -249,46 +249,32 @@ def test_prune_1():
     specs_to_remove = MatchSpec("numbapro"),
     with get_solver(specs_to_remove=specs_to_remove, prefix_records=final_state_1,
                     history_specs=specs) as solver:
-        unlink_precs, link_precs = solver.solve_for_diff(prune=False)
```

Comment:
prune is the default behavior.  This argument has no effect.  The prune=True case is shifted above, and remains similar.